replace_hash.py

```diff
@@ -0,0 +1,93 @@
+"""This scipt is used to replace specified placeholders(string) in a file(stored under ./output) with hashed random strings.
+Usage:
+      (bash) python3 replace_hash.py <filename>
+"""
+
+import re
+import random
+import string
+import sys
+from hashlib import sha256
+
+"""Read arguments in bash and store the filename in a variable.
+
+If filename not given in the bash code, program exit with error message.
+"""
+args = sys.argv
+if (len(args) < 2):
+  sys.exit("No filename given!")
+param_filename = args[1] 
+
+def in_file_replace(filename, hash_placeholders_list=[f"__HASH__PLACEHOLDER__", f"__SOME_HASH__"]):
+  """Replace specified placeholders in a file.
+
+  This function reads the contents of a file, store the content in a string, 
+      then pass the string to the 'replace_string' function to replace the placholders.
+      The modified string is then stroed back to the file. If no error shows up, 
+      the function prints a message of successfullly modofied.
+
+  Args:
+      filename (str): The name of the file (stored under ./output) to modify.
```

Comment:
I would say the argument is technically the `path` to the file, not really the file name, right?